src/lib.rs

```diff
@@ -84,6 +90,14 @@
 //! }
 //! ```
 //!
+//! # Ways of querying
+//!
+//! Tiberius offers two ways to query the database: directly from the [`Client`]
+//! with the [`Client#query`] and [`Client#execute`] methods when the query parameters are
+//! statically known in compile-time. Additinally one can construct a [`Query`]
```

Comment:
They're not necessarily statically known, I think you can get away with a dynamic `Vec<&dyn ToSql>`. The ergonomics issue here is the `&`.

Reply:
Going from `Vec<String>` to `&[&dyn ToSql]` is the issue. Or anything owned in a collection... Yeah this could be rephrased...